src/trampoline.c

```diff
@@ -53,9 +53,12 @@ plt_fn *create_trampoline(void *ctx, plt_fn *routine)
     mmk_assert(trampoline_sz < PAGE_SIZE);
 
 # if defined HAVE_MMAP_MAP_ANONYMOUS
+#  if !defined MAP_JIT
+#   define MAP_JIT 0
+#  endif
     void **map = mmap(NULL, PAGE_SIZE,
-            PROT_READ | PROT_WRITE | PROT_EXEC,
```

Comment:
Should we remove PROT_EXEC from the other mmap calls too? That would be slightly more consistent.